test/functional/3rd-party/3rd-party-bundle-list-deps.bats

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bats
+
+# Author: Castulo Martinez
+# Email: castulo.martinez@intel.com
+
+load "../testlib"
+
+global_setup() {
```

Comment:
Why are some tests added in the commit: "Add basic tests for 3rd-party bundle-list" and some tests are added in "Implementing 3rd-party bundle-list"?